lib/gui/index.html

```diff
@@ -49,5 +50,15 @@
         manifest-bind="version"
         os-open-external="https://github.com/resin-io/etcher/blob/master/CHANGELOG.md"></span>
     </footer>
+
+    <div class="section-loader"
+      ng-controller="StateController as state"
+      ng-class="{
+        isFinish: state.is('success')
+      }">
+      <safe-webview
+        src="'https://etcher.io/success-banner/'"
```

Comment:
Does this definitely need to be a single-quoted string inside a double-quoted string? What happens if somebody edits it to only be quoted by single-quotes or double-quotes?